sounddevice.py

```diff
@@ -2349,6 +2376,97 @@ def __init__(self, channel_selectors):
             channelSelectors=self._selectors))
 
 
+class MacCoreSettings(object):
+
+    def __init__(self, channel_map=None, change_dev_params=False, prevent_rate_conv=False, rate_conv_quality='Max'):
+        """Mac Core Audio-specific input/output settings.
+
+        Objects of this class can be used as *extra_settings* argument
+        to `Stream()` (and variants) or as `default.extra_settings`.
+
+        Parameters
+        ----------
+        channel_map : iterable of int, optional
+            Support for opening only specific channels of a MacCore
+            device.  *channel_map* is a list of integers specifying the
+            (zero-based) channel numbers to use.  Note that
+            *channel_map* is treated differently between input and
+            output channels.  See the example below.
+        change_dev_params : bool, optional
+            If ``True``, allows PortAudio to change things like the
+            device's frame size, which allows for much lower latency,
+            but might disrupt the device if other programs are using it,
+            even when you are just Querying the device.  ``False`` is
+            the default.
+        prevent_rate_conv : bool, optional
```

Comment:
Same here. If you want it to be shorter, you could use `prevent_conversion`.

I think in the rest of the API I tried to mimic the original PortAudio names, so why not `fail_if_conversion_required`?
It's not pretty, but at least it's not our fault!